find_max.py

```diff
@@ -0,0 +1,16 @@
+
+# Find the maximum element between index 0 and max_index, exclusive
+
+def find_max(array, max_index):
+    max = None
+    if max_index == 0:
```

Comment:
I think maybe you got your variables confused? I don't want to presume intent, but did you mean "len(array)" here rather than max_index?